src/App.js

```diff
@@ -1,14 +1,21 @@
 import React, { Component } from 'react'
 import IntroSection from './IntroSection.js'
 import InstaSection from './InstaSection.js'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
```

Comment:
What's the benefit of using flex in this scenario? If we removed lines 8 and 9, the layout would be the same, right?

Reply:
In order for an element to use flex rules that apply to the element rather than its children such as 
`justify-self, align-self ... etc` - these properties allow you to easily move elements around the page `(flex-start|center|flex-end)`
its parent needs to declare `display: flex` 

So, yes the layout would be the same, but you would lose the ability to control child elements with flex rules